CONTRIBUTING.md

```diff
@@ -28,6 +28,10 @@ issues first, as it's possible that someone else reported the same issue before
 you. Though it helps save time, don't worry! We won't mind if you accidentally
 post a duplicate issue.
 
+Please note that if an issue can only be reproduced on a nightly compiler it will be closed and marked "Will not fix."
```

Comment:
Err, this doesn't seem very welcome / may cause more problems later. I mean, if somebody experiences an issue, it may be unknown to rust-lang/rust yet, so we should report it. I'd therefore just say that they should try stable, but still create an issue if it fails on nightly only.